seqr/views/utils/project_context_utils.py

```diff
@@ -132,20 +133,20 @@ def families_discovery_tags(families):
 MME_TAG_NAME = 'MME Submission'
 
 
-def add_project_tag_types(projects_by_guid, add_counts=False):
-    variant_tag_types_models = VariantTagType.objects.filter(Q(project__guid__in=projects_by_guid.keys()) | Q(project__isnull=True))
-    variant_tag_types = _get_json_for_models(variant_tag_types_models)
+def add_project_tag_types(projects_by_guid, project=None):
+    is_single_project = len(projects_by_guid) == 1
+    project_q = dict(project=project) if project else dict(project__guid__in=projects_by_guid.keys())
+    variant_tag_types_models = VariantTagType.objects.filter(Q(**project_q) | Q(project__isnull=True))
+    variant_tag_types = get_json_for_queryset(
```

Comment:
`get_json_for_queryset` is more efficient than `_get_json_for_models`, I just haven't gotten around to fully replacing it yet